lib/net/peer.js

```diff
@@ -2145,6 +2185,16 @@ class PeerOptions {
       this.banScore = options.banScore;
     }
 
+    if (options.dosScore != null) {
+      assert(typeof options.dosScore === 'number');
+      this.dosScore = options.dosScore;
+    }
+
+    if (options.p2prps != null) {
```

Comment:
This should be parsed from the config and passed through in the file https://github.com/handshake-org/hsd/blob/master/lib/node/fullnode.js

Reply:
OK, added a config option `p2prps`. Could be worded and explained better imo since it only affects getproof messages atm.